cocotb_test/simulator.py

```diff
@@ -268,7 +268,7 @@ def execute(self, cmds):
             self.process = None
             if p.returncode:
                 self.logger.error("Command terminated with error %d" % p.returncode)
```

Comment:
Maybe remove line:
```
self.logger.error("Command terminated with error %d" % p.returncode)
```

Reply:
I don't know if this is needed to make sure something else somewhere keeps working. But sure, I can remove it...

Reply:
Can you remove it? It looks OK now.

Reply:
Done. You are welcome to make these kind of changes yourself, of course.